### PR TITLE
JSON parse and stringify examples and explanation with an open endpoint

### PR DESCRIPTION
**Pull request by EddyVinck**
Fixes https://github.com/Swap76/Learn-JavaScript/issues/30

### CODE REVIEW

File: JavaScript_Advance/jsonParseAndStringify.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// We use the cors-anywhere domain to circumvent CORS errors for demo purposes
+// You can read more about it here: https://github.com/Rob--W/cors-anywhere/
+// This allows you to just copy the code here and paste it in a JavaScript console in your browser.
+const cors_api_url = "https://cors-anywhere.herokuapp.com/";
```
Comment (Swap76) [inline JavaScript_Advance/jsonParseAndStringify.js]: Good work of talking about cors 

super helpful for others

**Comment by EddyVinck** [comment]
Thanks guys! ✌️

**Comment by Swap76** [comment]
@EddyVinck  It was great working with you.

Looking forward to working with you :-)

Star to the repo is highly appreciated.

**Comment by Swap76** [comment]
@EddyVinck The work you have done for the project is highly appreciated. Thanks :-)

Looking forward to working with you on my new project related to Deno https://github.com/Swap76/Learn-Deno/ and together making this better.

Star to the repo is highly appreciated.

**Pull request merged**